helm-charts/azure-api-management-gateway/values.yaml

```diff
@@ -14,6 +14,8 @@ nameOverride: ""
 fullnameOverride: ""
 
 gateway:
+  deployment:
+    terminationGracePeriodSeconds: 60
```

Comment:
Let's not define a default
```suggestion
    terminationGracePeriodSeconds: 
```

Reply:
terminationGracePeriodSeconds is set to 60 in the yaml that is generated from the portal. Shouldn't it be the same?

Reply:
Not specifically, I think it's best to stick with Kubernetes defaults here